src/arc.rs

```diff
@@ -6,10 +6,16 @@ use std::f64::consts::{FRAC_PI_2, PI};
 /// A single arc segment.
 #[derive(Clone, Copy, Debug)]
 pub struct Arc {
+    /// The arc's centre point.
```

Comment:
I believe this is right, but it's no longer fresh in my mind, and the logic was adapted from a number of sources (including lyon).